migrations/240_pt_tables__create.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS pt_exams (
+    id bigserial PRIMARY KEY,
+    uuid uuid
+);
+
+CREATE TABLE IF NOT EXISTS pt_enrollments (
+    id BIGSERIAL PRIMARY KEY,
+    user_id BIGINT NOT NULL REFERENCES users
+);
+
+CREATE TABLE IF NOT EXISTS pt_reservations (
+    id bigserial PRIMARY KEY,
+    exam_id BIGINT NOT NULL REFERENCES pt_exams,
+    enrollment_id BIGINT NOT NULL REFERENCES pt_enrollments,
+    access_start TIMESTAMPTZ,
+    access_end TIMESTAMPTZ
+);
```

Comment:
I believe that this migration and the "same" migration in PT will interact poorly if we deploy PL to a database and later try to deploy PT on top of it. I understand this is not a concern at the moment, but something to keep in mind!

See https://github.com/PrairieLearn/PrairieSchedule/blob/7c5ccd51d0a1c8fcabe38745fba564443bd0fefb/migrations/000_initial_state.sql#L81 for an example of how PS deals with this.

I have vague thoughts about changing the way we do migrations to avoid this problem entirely, inspired by [this GitHub blog post](https://github.blog/2020-02-14-automating-mysql-schema-migrations-with-github-actions-and-more/), but they are very much just thoughts at this point 😅

Reply:
Ha! I was also thinking about this exact same problem! I have a much hackier solution in mind than what GitHub did 😄. But as you say, we can figure this out later.